### DISCUSSION TITLE
Filter releases

### DISCUSSION THREAD

**Discussion by andrew-hwahin** [microsoft/fluentui#17024] [Q&A] [answered]
How do I filter the releases to only show the package of @fluentui/react?
![image](https://user-images.githubusercontent.com/61768800/108176371-e93d0980-713c-11eb-9070-605237905892.png)
![image](https://user-images.githubusercontent.com/61768800/108175729-10dfa200-713c-11eb-9e86-faeaa457d661.png)

**Comment by khmakoto** [comment, marked as answer]
Right now it is kind of impossible to do that in GitHub.

The workflow that works best for me is to look at available versions at [npmjs](https://www.npmjs.com/package/@fluentui/react) and then just replace whatever version number in `https://github.com/microsoft/fluentui/releases/tag/%40fluentui%2Freact_v[`_`Version number`_`]`, so, if you want to look at version 7.160.2 for example you'd do [https://github.com/microsoft/fluentui/releases/tag/%40fluentui%2Freact_v7.160.2](https://github.com/microsoft/fluentui/releases/tag/%40fluentui%2Freact_v7.160.2).

I hope that helps!
